projects/packages/paypal-payments/src/legacy/class-simple-payments.php

```diff
@@ -0,0 +1,831 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
```

Comment:
I think you should be able to remove this now that the name matches the class?

Reply:
Thanks for catching this. I've remove it in [a784c47](https://github.com/Automattic/jetpack/pull/43413/commits/a784c47e2e8765fedab3d193f202ccbd160cc6dc).